index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>The Weatherly</title>
+        <link href="./styles/index.css" rel="stylesheet" > 
+    </head>
+
+    <body>
+        <section>
+            <header>
+                <h1>The Weather For</h1>
+                <span id="display"> &#x1f3d9; Seattle &#x1f3d9; </span>
+            </header>
+        </section>
+        <main>
+            <div>
+                <section>
+                    <h3>Temperature</h3>
```

Comment:
It looks like you skipped straight from `h1` to `h3`, remember, these tags should work to build an outline of your site:

```suggestion
                    <h2>Temperature</h2>
```

If the text is too big you can always change the size in CSS.